osxkeychain.go

```diff
@@ -478,3 +554,49 @@ func createAccess(label string, trustedApplications []string) (C.SecAccessRef, e
 
 	return access, nil
 }
+
+func CreateKeychain(path string, password string) error {
+	passwordRef := C.CString(password)
+	defer C.free(unsafe.Pointer(passwordRef))
+
+	pathName := C.CString(path)
+	defer C.free(unsafe.Pointer(pathName))
+
+	// without passing in kref we get 'One or more parameters passed to a function were not valid.'
```

Comment:
Whoa, really? The docs explicitly say that nil can be passed in. May want to mention that.